testsuite/MDAnalysisTests/data/coordinates/test_incomplete_vel.gro

```diff
@@ -0,0 +1,8 @@
+Written by MDAnalysis
+    5
+    1MET     CA    1   0.000   0.100   0.200  0.0000  0.0100  0.0200
+    2ARG     CA    2   0.300   0.400   0.500
```

Comment:
These broken files are extremely common. They happen when you build a system by stitching pieces together. T he warning seems a good idea; for sure we should be able to read the files.